src/test/java/com/ibm/watson/modelmesh/ModelMeshTearDownTest.java

```diff
@@ -315,7 +315,7 @@ public static int getPID(Process process) throws Exception {
     }
 
     public static int killProcess(Process process) throws Exception {
-        int pid = getPID(process);
-        return Runtime.getRuntime().exec("kill -9 " + pid).waitFor();
+        String pid = Integer.toString(getPID(process));
```

Comment:
this change should be fine although `pid` cannot be entered (or manipulated) by a malicious user (and it's test only code)